eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -28,6 +28,14 @@ stages:
                 deploy:
                   steps:
                     - checkout: self
+                    - script: |
```

Comment:
What in our verify-changelog requires the ci_tools to be pip installed first?

Reply:
That's not needed since we are no longer using the previous changelog verification. @praveenkuttappan is there any other why you added this.